<launch>
    <arg name="marker_length" default="0.04" />
    <arg name="aruco_target_id" default="[1, 4]" />
    <arg name="scan_limit" default="1" />

    <node name="TableDetector" pkg="table_detector" type="TableDetector.py" output="screen">
        <param name="marker_length" value="$(arg marker_length)" />
        <param name="aruco_target_id" value="$(arg aruco_target_id)" />
        <param name="scan_limit" value="$(arg scan_limit)" />
        <rosparam file="$(find table_detector)/config/master_calibration.yaml" />
    </node>
</launch>